.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+    tags:
+      - v*
+  pull_request:
+    branches: [ master ]
+    tags: 
+      - v*
```

Comment:
Probably want to separate the tagged on master builds from the pull request builds — so that it doesn't try to deploy releases on pull requests.

Reply:
I agree